dapp/src/partials.js

```diff
@@ -476,7 +476,7 @@ angular.module('multiSigWeb').run(['$templateCache', function($templateCache) {
     "      Payout Bounty\n" +
     "    </button>\n" +
     "    <button type=\"button\" class=\"btn btn-primary btn-lg\" disabled-if-no-accounts\n" +
-    "      ng-click=\"withdrawToken(wallet.tokens['0x6b175474e89094c44da98b954eedeac495271d0f'])\">\n" +
+    "      ng-click=\"withdrawToken(wallet.tokens['0x6B175474E89094C44Da98b954EedeAC495271d0F'])\">\n" +
```

Comment:
I was wondering, why we are not using address from config file? 🤔
Same applies to line 498 and changes inside `partials/wallet`

Reply:
Just because it was a quick hack :)